### PR TITLE
solve issues #17...19

### PR DESCRIPTION
**Pull request by berndfinger**

### CODE REVIEW

**Review by richm — APPROVED** [review]
lgtm

**Review by richm — APPROVED** [review]
lgtm

File: tasks/sapnote/assert-2526952.yml [hunk 1]
```diff
@@ -3,32 +3,28 @@
...
   assert:
     that:
-      - "'active (running)' in tuned_status.stdout"
+      - ansible_facts.services['tuned.service']['state'] == 'running'
     fail_msg: "FAIL: Service tuned is NOT active!"
     success_msg: "PASS: Service tuned is active."
   ignore_errors: "{{ sap_netweaver_preconfigure_assert_ignore_errors|d(false) }}"
 
-- name: Report tuned-profiles-sap version
+- name: Report tuned-profiles-sap-hana version
```
Comment (richm) [inline tasks/sapnote/assert-2526952.yml:34]: sap or sap-hana?

**Review by rhmk — CHANGES_REQUESTED** [review]
lgtm

File: tasks/sapnote/assert-2526952.yml (hunk 1 above)
Comment (berndfinger) [reply tasks/sapnote/assert-2526952.yml]: thanks

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**